requirements.txt

```diff
@@ -8,7 +8,7 @@ argparse==1.2.1
 boto==2.35.1
 flask-mongoengine==0.7.1
 mongoengine==0.10.0
-mongomock==3.2.1
+git+https://github.com/OriHoch/mongomock.git@support-for-operator-filtering-inside-list-fields#egg=mongomock
```

Comment:
my mongomock branch contains a fix for searching in list fields (for the marriage_years)